public/video-ui/src/components/ReactApp.js

```diff
@@ -73,8 +73,8 @@ function mapStateToProps(state) {
 
 function mapDispatchToProps(dispatch) {
   return {
-    appActions: bindActionCreators(Object.assign({}, updateSearchTerm, getVideo, publishVideo), dispatch)
+    appActions: bindActionCreators(Object.assign({}, updateSearchTerm, getVideo, publishVideo, saveVideo), dispatch)
```

Comment:
the linter was complaining about an unused import here - https://github.com/guardian/media-atom-maker/pull/231/files#diff-1fd58d0340aed2bb63a38c5f92e9eb65R63

I'll release to CODE before merging to make sure I've not broken anything.